### PR TITLE
Feat/banner loading

### PR DESCRIPTION
**Pull request by myo3**

### CODE REVIEW

**Comment by thomasmso** [comment]
Can we remove the event tracking stuff from here? I understand that it's pretty trivial, but I think it's a good habit to have branches/PRs strictly have features/fixes/whatever it is concerned with.

File: AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/LayoutEditorBannerActivity.java [hunk 1]
```diff
@@ -26,12 +28,21 @@ protected void onCreate(final Bundle savedInstanceState)
 
         adStatusTextView = (TextView) findViewById( R.id.status_label );
 
+        Button loadButton = (Button) findViewById(R.id.load_button);
```
Comment (christophercong) [inline AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/LayoutEditorBannerActivity.java]: Let's mark this as `final`.

File: AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/MainActivity.java [hunk 2]
```diff
@@ -39,8 +40,9 @@
     private static final int POSITION_INCENTIVIZED  = 1;
     private static final int POSITION_NATIVE        = 2;
     private static final int POSITION_BANNER        = 3;
-    private static final int POSITION_RESOURCES     = 5;
-    private static final int POSITION_CONTACT       = 6;
+    private static final int POSITION_EVENT         = 5;
```
Comment (thomasmso) [inline AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/MainActivity.java]: I don't think this belong in this PR.

File: AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/MainActivity.java [hunk 3]
```diff
@@ -49,6 +51,7 @@
             new ListItem( "Native Ads", "In-content ads that blend in seamlessly" ),
             new ListItem( "Banners", "320x50 Classic banner ads" ),
             new ListItem( "MRecs", "Please reference banners demo" ),
+            new ListItem( "Event Tracking", "Track in-app events for your users"),
```
Comment (thomasmso) [inline AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/MainActivity.java]: I don't think this belong in this PR.

File: AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/MainActivity.java [hunk 4]
```diff
@@ -130,6 +133,11 @@ else if ( position == POSITION_BANNER )
                     Intent intent = new Intent( MainActivity.this, BannerListViewActivity.class );
                     startActivity( intent );
                 }
+                else if ( position == POSITION_EVENT )
```
Comment (thomasmso) [inline AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/MainActivity.java]: I don't think this belong in this PR.

File: AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/LayoutEditorBannerActivity.java [hunk 5]
```diff
@@ -26,12 +28,21 @@ protected void onCreate(final Bundle savedInstanceState)
 
         adStatusTextView = (TextView) findViewById( R.id.status_label );
 
+        final Button loadButton = (Button) findViewById(R.id.load_button);
```
Comment (thomasmso) [inline AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/LayoutEditorBannerActivity.java]: Could you please adhere to AppLovin coding styles?

File: AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/LayoutEditorBannerActivity.java [hunk 6]
```diff
@@ -26,12 +28,21 @@ protected void onCreate(final Bundle savedInstanceState)
...
+
         // Retrieve banner from layout editor
         final AppLovinAdView adView = (AppLovinAdView) findViewById( R.id.ad_view );
 
         //
         // Optional: Set listeners
         //
+        loadButton.setOnClickListener(new View.OnClickListener() {
```
Comment (thomasmso) [inline AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/LayoutEditorBannerActivity.java]: Can you move this to after L31 where the `loadButton` was referenced?

```
//
// Optional: Set listeners
//
```

is referring to listeners for the `adView`, not the `loadButton`.

File: AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/ProgrammaticBannerActivity.java [hunk 7]
```diff
@@ -23,28 +25,40 @@
...
     @Override
-    protected void onCreate(final Bundle savedInstanceState)
+    protected void onCreate( final Bundle savedInstanceState )
     {
         super.onCreate( savedInstanceState );
         setContentView( R.layout.activity_banner_programmatic );
 
         adStatusTextView = (TextView) findViewById( R.id.status_label );
 
+        // Retrieve banner from layout editor
```
Comment (thomasmso) [inline AppLovin-Android-Demo-App/src/main/java/com/applovin/demoapp/banners/ProgrammaticBannerActivity.java]: I'm pretty sure you are not retrieving the banner from the Layout Editor here

**Pull request merged**